src/Models/Thread.php

```diff
@@ -121,6 +121,30 @@ public static function getAllLatest()
         return static::latest('updated_at');
     }
 
+    /**
+     * Thread relationship.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     *
+     * @codeCoverageIgnore
```

Comment:
Why do you ignore this method in coverage?

Reply:
I had just copied it from a neighboring method, not sure what the reasoning for that tag is

Reply:
ah yes I missed that cleanup, i'll fix